identity-apps-core/components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/listner/AppPortalRoleManagementListener.java

```diff
@@ -100,6 +102,15 @@ public void preUpdateUserListOfRole(String roleId, List<String> newUserIDList, L
             return;
         }
 
+        try {
+            if (isRoleModificationAllowedForTenant(tenantDomain)) {
+                return;
+            }
+        } catch (OrganizationManagementException e) {
+            throw new IdentityRoleManagementException(INVALID_REQUEST.getCode(),
```

Comment:
Can we say that if the `OrganizationManagementException` occurred, it is an invalid request. In the high level it seems like this exception can be occurred from internal processes.

Reply:
What if we remove the Invalid code from it? Similar to the UserStoreException in the try-catch below?

Reply:
Let's add the relevant details.

Reply:
This is now updated